### PR TITLE
update next-release-preview/index.md

### PR DESCRIPTION
**Pull request by kirangodishala**
A brief description of clouddriver sharding is added.

### CODE REVIEW

**Comment by aimeeu** [comment]
@kirangodishala Can you provide the link to the code PR that implements this functionality?

**Comment by kirangodishala** [comment]
> @kirangodishala Can you provide the link to the code PR that implements this functionality?

https://github.com/spinnaker/clouddriver/pull/5295

Do you want me to add this in the index.md?

File: community/releases/next-release-preview/index.md [hunk 1]
```diff
@@ -12,3 +12,8 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 1.26
+
+### Clouddriver account sharding for caching improvements
+
+The sql backed clouddriver caching pods are now capable of sharding the accounts of any provider 
+by setting a new configuration property `caching.sharding-enabled` to `true`. A hashing logic is applied to determine which pod can run the caching agents of an account.
```
Comment (german-muzquiz) [inline community/releases/next-release-preview/index.md]: I'd add that this feature is opt-in and disabled by default, it only works when using specifically the SQL backend scheduler (as you can have cache stored in SQL but still use the redis scheduler), and it works for all cloud providers by splitting accounts based on their name, so all caching agents for the same account are executed in the same pod, but not all pods will cache all accounts.

Comment (kirangodishala) [reply community/releases/next-release-preview/index.md]: Will update accordingly.

**Review by aimeeu — APPROVED** [review]

**Review by german-muzquiz — APPROVED** [review]

**Pull request merged**